pkg/serviceaccount/serviceaccounts_controller.go

```diff
@@ -42,23 +43,27 @@ func nameIndexFunc(obj interface{}) (string, error) {
 }
 
 type ServiceAccountControllerOptions struct {
```

Comment:
there are actually two controllers in the package, one for managing service accounts and one for managing tokens... hence the qualifier